src/telemetry/unhandledErrorReporter.ts

```diff
@@ -15,9 +15,20 @@ export const enum ErrorType {
 export function installUnhandledErrorReporter(
   logger: ILogger,
   telemetryReporter: ITelemetryReporter,
+  isVsCode?: boolean,
```

Comment:
Can you make this required, or default to `true` in order to be conservative about the data we send?

Reply:
done